### PR TITLE
Mention how inline event handlers are synthesized

### PR DESCRIPTION
**Pull request by Josh-Cena**
#### Motivation

While looking at https://github.com/mdn/content/issues/5578, I also became curious about _what_ constitutes a valid event handler attribute. However, this information is documented absolutely nowhere, and I had to read the HTML spec instead. I think this is worth two paragraphs.

Also want to mention that, with the recent `GlobalEventHandlers` demixing effort, the `onxxx` handlers are becoming increasingly hard to search. I literally searched for `onclick` and the only thing that came up was SVG `onclick` attribute which is not useful—even misleading—to users. I don't even know what to link to in these pages.

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/learn/javascript/building_blocks/events/index.md [hunk 1]
```diff
@@ -296,7 +296,7 @@ function bgChange() {
 }
 ```
 
-The earliest method of registering event handlers found on the Web involved **event handler HTML attributes** (or **inline event handlers**) like the one shown above — the attribute value is literally the JavaScript code you want to run when the event occurs.
+The earliest method of registering event handlers found on the Web involved [**event handler HTML attributes**](/en-US/docs/Web/HTML/Attributes#event_handler_attributes) (or **inline event handlers**) like the one shown above — the attribute value is literally the JavaScript code you want to run when the event occurs.
```
Comment (teoli2003) [inline files/en-us/learn/javascript/building_blocks/events/index.md:299]: nit
```suggestion
The earliest method of registering event handlers found on the Web involved [_event handler HTML attributes_](/en-US/docs/Web/HTML/Attributes#event_handler_attributes) (or _inline event handlers_) like the one shown above — the attribute value is literally the JavaScript code you want to run when the event occurs.
```

File: files/en-us/web/html/global_attributes/index.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ In addition to the basic HTML global attributes, the following global attributes
...
+- The [event handler](/en-US/docs/Web/HTML/Attributes#event_handler_attributes) attributes: **`onabort`**, **`onautocomplete`**, **`onautocompleteerror`**, **`onblur`**, **`oncancel`**, **`oncanplay`**, **`oncanplaythrough`**, **`onchange`**, **`onclick`**, **`onclose`**, **`oncontextmenu`**, **`oncuechange`**, **`ondblclick`**, **`ondrag`**, **`ondragend`**, **`ondragenter`**, **`ondragleave`**, **`ondragover`**, **`ondragstart`**, **`ondrop`**, **`ondurationchange`**, **`onemptied`**, **`onended`**, **`onerror`**, **`onfocus`**, **`oninput`**, **`oninvalid`**, **`onkeydown`**, **`onkeypress`**, **`onkeyup`**, **`onload`**, **`onloadeddata`**, **`onloadedmetadata`**, **`onloadstart`**, **`onmousedown`**, **`onmouseenter`**, **`onmouseleave`**, **`onmousemove`**, **`onmouseout`**, **`onmouseover`**, **`onmouseup`**, **`onmousewheel`**, **`onpause`**, **`onplay`**, **`onplaying`**, **`onprogress`**, **`onratechange`**, **`onreset`**, **`onresize`**, **`onscroll`**, **`onseeked`**, **`onseeking`**, **`onselect`**, **`onshow`**, **`onsort`**, **`onstalled`**, **`onsubmit`**, **`onsuspend`**, **`ontimeupdate`**, **`ontoggle`**, **`onvolumechange`**, **`onwaiting`**.
```
Comment (teoli2003) [inline files/en-us/web/html/global_attributes/index.md:22]: I don't see the use of the bold here (and as we edit this line 😉 ):
```suggestion
- The [event handler](/en-US/docs/Web/HTML/Attributes#event_handler_attributes) attributes: `onabort`, `onautocomplete`, `onautocompleteerror`, `onblur`, `oncancel`, `oncanplay`, `oncanplaythrough`, `onchange`, `onclick`, `onclose`, `oncontextmenu`, `oncuechange`, `ondblclick`, `ondrag`, `ondragend`, `ondragenter`, `ondragleave`, `ondragover`, `ondragstart`, `ondrop`, `ondurationchange`, `onemptied`, `onended`, `onerror`, `onfocus`, `oninput`, `oninvalid`, `onkeydown`, `onkeypress`, `onkeyup`, `onload`, `onloadeddata`, `onloadedmetadata`, `onloadstart`, `onmousedown`, `onmouseenter`, `onmouseleave`, `onmousemove`, `onmouseout`, `onmouseover`, `onmouseup`, `onmousewheel`, `onpause`, `onplay`, `onplaying`, `onprogress`, `onratechange`, `onreset`, `onresize`, `onscroll`, `onseeked`, `onseeking`, `onselect`, `onshow`, `onsort`, `onstalled`, `onsubmit`, `onsuspend`, `ontimeupdate`, `ontoggle`, `onvolumechange`, `onwaiting`.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Two nits.

File: files/en-us/web/html/global_attributes/index.md (hunk 2 above)
Comment (Josh-Cena) [reply files/en-us/web/html/global_attributes/index.md:22]: I assume we would need to update the other line as well?

Comment (teoli2003) [reply files/en-us/web/html/global_attributes/index.md]: Yes, please.

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**